packages/shared/src/components/header/BreadCrumbs.tsx

```diff
@@ -1,31 +1,44 @@
 import React, { PropsWithChildren, ReactElement } from 'react';
+import { useRouter } from 'next/router';
 import classed from '../../lib/classed';
 import { Button, ButtonSize, ButtonVariant } from '../buttons/Button';
 import { HomeIcon } from '../icons';
 import { WithClassNameProps } from '../utilities';
 
 const BreadCrumbsWrapper = classed(
-  'div',
-  'hidden h-10 gap-0.5 items-center p-1.5 text-surface-secondary laptop:flex',
+  'nav',
+  'hidden h-10 gap-0.5 items-center px-1.5 text-surface-secondary laptop:flex',
 );
 
 export const BreadCrumbs = ({
   children,
   className,
 }: PropsWithChildren<WithClassNameProps>): ReactElement => {
+  const { pathname } = useRouter();
+
   return (
-    <BreadCrumbsWrapper className={className}>
-      <Button
-        variant={ButtonVariant.Tertiary}
-        icon={<HomeIcon secondary />}
-        tag="a"
-        href={process.env.NEXT_PUBLIC_WEBAPP_URL}
-        size={ButtonSize.XSmall}
-      />
-      <span>/</span>
-      <div className="flex flex-row items-center gap-1 px-2 font-bold text-text-primary typo-callout">
-        {children}
-      </div>
+    <BreadCrumbsWrapper aria-label="breadcrumbs" className={className}>
+      <ol className="flex-1 items-center gap-0.5 laptop:flex">
+        <li className="flex flex-row items-center gap-0.5">
+          <Button
+            variant={ButtonVariant.Tertiary}
+            icon={<HomeIcon secondary />}
+            tag="a"
+            href={process.env.NEXT_PUBLIC_WEBAPP_URL}
+            size={ButtonSize.XSmall}
+          />
+          <span aria-hidden>/</span>
+        </li>
+        <li className="flex flex-row items-center gap-1 px-2 font-bold text-text-primary">
+          <a
+            href={pathname}
```

Comment:
Shouldn't the current one not be a link as it will just reload?

Reply:
I've put the `pointer-events:none;` for not making it clickable, but should be present for semantic. 
But this is optional, I can remove it if this make troubles!

Reply:
Not sure just seems pointless then right?
Why is it semantic needed if it doesn't do anything?

Reply:
Just for having all the `li` with a `a`, and keeping them consistent through screen readers. But I'm removing since is optional 👍